internal/clientconn/conn.go

```diff
@@ -441,6 +442,24 @@ func (c *conn) route(connCtx context.Context, reqHeader *wire.MsgHeader, reqBody
 
 		command = document.Command()
 
+		if comment, cerr := document.Get("comment"); cerr == nil {
+			if cmt, ok := comment.(string); ok {
+				var spanCtx trace.SpanContext
+
+				spanCtx, cerr = spanContextFromComment(cmt)
+				if cerr != nil {
+					c.l.ErrorContext(connCtx, "Failed to extract span context from comment", logging.Error(err))
```

Comment:
Do you think error level is too high for this log? Users can set string comment and that's expected behaviour.